placetobe-header.html

```diff
@@ -165,7 +165,7 @@
       <nav>
         <a href="/"><placetobe-logo></placetobe-logo></a>
         <ul>
-          <!-- <li hidden><a href="/campagnes" data-path="campagnes" data-active$="[[investPageActive]]">Campagnes</a></li> -->
+          <li hidden="[[campaignArrayHidden]]"><a href="/campagnes" data-path="campagnes" data-active$="[[campaignPageActive]]">Campagnes</a></li>
```

Comment:
zullen we dit `[[!hasCampaigns]]` noemen? is logischer naam

Reply:
done